docs/user_guide/getting_started.md

```diff
@@ -182,6 +182,7 @@ our [service profile documentation](config.md#available-profiles) for details.
 - Try the Neurobagel node you just deployed by accessing:
     - your own query tool at  [http://localhost:3000](http://localhost:3000), and reading the [query tool usage](./query_tool.md#usage) guide
     - the interactive docs for your node API at [http://localhost:8000/docs](http://localhost:8000/docs), and reading the [API usage](./api.md) guide
+- Change the [default ports of services](config.md#default-ports-for-services)
```

Comment:
just for my understanding: as opposed to what was originally requested in the issue, we're intentionally phrasing this as "**changing** the default ports" rather than something like "open service ports on your machine" because we want to discourage users from doing that in general unless absolutely necessary, correct?

Reply:
Yes. Someone who wants to open the ports in a firewall will likely know to look there as well